### ISSUE TITLE
Version 0.13 - modal does not close correctly

### ISSUE THREAD

**Issue by djvs** [angular-ui/bootstrap#3873]
Backdrop still remains and modal itself is still there, just with opacity of 0.  Don't think this is a duplicate of https://github.com/angular-ui/bootstrap/issues/3633 .    Does not occur with 0.12.

**Comment by djvs** [comment]
Disabling animation on a modal eliminates the bug.

**Comment by wesleycho** [comment]
Can you create a reproduction with [Plunker](http://plnkr.co/)?

**Comment by esion** [comment]
I got this issue too after migrate to angular 1.4

unfortunatly I can't reproduce on plkr : http://plnkr.co/edit/xofiwx?p=preview  
I need to investigate a bit more

**Comment by esion** [comment]
sorry I didn't read others issues, I suppose this is more complicated than just an angular upgrade

**Comment by esion** [comment]
> TL;DR: In order to fix : set animate: false or remove angular-animate from dependencies

Since there is no version specified in this issue, I use angular 1.4.

This is what I found : 

I've updated http://plnkr.co/edit/xofiwx?p=preview to make backdrop not removed
it only requires to add angular-animate

in ui-bootstrap we use the event `$animate:close` which is never fired so the backdrop is never removed from dom : https://github.com/angular-ui/bootstrap/blob/master/src/modal/modal.js#L238

the actual doc for `angular.element(selector').one()` says it doesn't support namespaces I m not sure if it is related.

Hope this help to fix.

**Comment by djvs** [comment]
Yes, thank you esion, wish I caught this 20 minutes ago.  Made a sad plunker which doesn't duplicate it, http://plnkr.co/edit/NCmCk722weDma3Mj2b9m?p=preview.

**Comment by djvs** [comment]
However removing ng-animate doesn't seem to resolve it for me, while it does on your plunker...

**Comment by djvs** [comment]
Ah got it - ngMaterial will *also* conflict and produce this issue.

**Comment by djvs** [comment]
AND this...https://github.com/homerjam/angular-ui-router-anim-in-out

**Comment by esion** [comment]
I see ui-router-anim requires ngAnimate. 
But no idea for ngMaterial

**Comment by esion** [comment]
So I think this the same discussion as in #3633 so this one should be closed

**Comment by djvs** [comment]
If you look at Daniel's plunker, the behavior is different.

**Comment by esion** [comment]
He said it has been resolved by adding ngAnimate
https://github.com/angular-ui/bootstrap/issues/3633#issuecomment-106861328

Without it,  ui-bootstrap has no idea when remove modal from DOM. So it removes it asap on 1.3. It's just a "bug" in documentation not in code.

In case of angular 1.4, BC breaks have been added arround ngAnimate, so the event used to remove backdrop and modal window from DOM is not triggered anymore see L.238 as I mentionned, this has been discussed here : https://github.com/angular-ui/bootstrap/issues/3633#issuecomment-113516753

(Obviously there is lot of mess in the issue :p and two "bugs")
